changelogs/fragments/8178-redfish-info-multi.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
+  - redfish_info - modify ``GetLogs`` and ``GetNetworkProtocols`` commands to support multi-manager node types. These commands will return a list of dictionaries similar to the ``GetManagerInventory`` and ``GetSystemInventory`` commands instead of a single dictionary (https://github.com/ansible-collections/community.general/issues/6764).
```

Comment:
```suggestion
  - redfish_info - modify ``GetLogs`` and ``GetNetworkProtocols`` commands to support multi-manager node types. These commands will return a list of dictionaries similar to the ``GetManagerInventory`` and ``GetSystemInventory`` commands instead of a single dictionary (https://github.com/ansible-collections/community.general/issues/6764, https://github.com/ansible-collections/community.general/pull/8178).
```